kf_lib_data_ingest/common/file_retriever.py

```diff
@@ -224,10 +280,26 @@ def get(self, url, auth=None):
                     dir=self.storage_dir,
                     delete=self.cleanup_at_exit and not self.__tmpdir
                 )
+
+                # Validate auth configs dict if it exists
+                # Select one auth config based inspection of url
+                auth_config = None
```

Comment:
Instead of `auth_config = None` here, move it back to the method def as an optional input argument and check `if self.auth_configs and not auth_config` on the next line to preserve the ability to manually pass in a custom auth object for a request.

Reply:
Ok, I removed that stuff bc I thought we didn't want that. I can put it back